src/main/java/com/minecolonies/coremod/entity/citizen/citizenhandlers/CitizenChatHandler.java

```diff
@@ -44,33 +40,15 @@ public void notifyDeath(final DamageSource damageSource)
     {
         if (citizen.getCitizenColonyHandler().getColony() != null && citizen.getCitizenData() != null)
         {
-            final IJob<?> job = citizen.getCitizenJobHandler().getColonyJob();
-            MutableComponent contentComponent;
-            if (job != null)
-            {
-                contentComponent = Component.translatable(
-                  TranslationConstants.WORKER_DIED,
-                  Component.translatable(job.getJobRegistryEntry().getTranslationKey()),
-                  citizen.getCitizenData().getName(),
-                  Math.round(citizen.getX()),
-                  Math.round(citizen.getY()),
-                  Math.round(citizen.getZ()),
-                  Component.translatable(damageSource.msgId));
-            }
-            else
-            {
-                contentComponent = Component.translatable(
-                  TranslationConstants.COLONIST_DIED,
-                  citizen.getCitizenData().getName(),
-                  Math.round(citizen.getX()),
-                  Math.round(citizen.getY()),
-                  Math.round(citizen.getZ()),
-                  Component.translatable(damageSource.msgId));
-            }
-
-            MessageUtils.format(contentComponent)
-              .with(ChatFormatting.RED)
-              .sendTo(citizen.getCitizenColonyHandler().getColony()).forManagers();
+            MessageUtils.format("")
```

Comment:
There is an overload of the format function that directly accepts a component, so you can start with `MessageUtils.format(citizen.getCombatTracker().getDeathMessage())`

Reply:
I tried that, but that had weird results with the formatting (not everything was red then). So I just went for this, and it works(tm).